notes-service/src/main/java/org/exoplatform/wiki/jpa/JPADataStorage.java

```diff
@@ -38,8 +39,14 @@
 import org.exoplatform.container.xml.ValuesParam;
 import org.exoplatform.portal.config.*;
 import org.exoplatform.portal.config.model.PortalConfig;
+import org.exoplatform.services.security.ConversationState;
```

Comment:
As explained in https://docs.google.com/spreadsheets/d/1s-op62uot1dy4YwMA-JswxNg0DwS4sdmg1eAll0VHIU/edit#gid=0, the rule DEVSTD-28 says : 

> Avoid using org.exoplatform.services.security.ConversationState.getCurrent() in Service & Storage layers. In fact, the access to the currently connected user should be done from higher layer such as REST Service / Portlet. 

So we should no get user ID in this layer.